buffer/src/main/java/io/netty/buffer/UnpooledUnsafeHeapByteBuf.java

```diff
@@ -29,6 +29,10 @@
         super(alloc, initialCapacity, maxCapacity);
     }
 
+    byte[] allocateArray(int initialCapacity) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1161)